management/src/clusterm/manager/client_test.go

```diff
@@ -89,53 +100,59 @@ func (s *managerSuite) TestPostSuccess(c *C) {
 		url: baseURL,
 	}
 
+	var reqHostGroupBody bytes.Buffer
+	c.Assert(json.NewEncoder(&reqHostGroupBody).Encode(testReq1HostGroupBody), IsNil)
+
+	var testFlags ActionFlags
 	tests := map[string]struct {
 		expURLStr string
 		nodeName  string
-		extraVars string
+		flags     ActionFlags
 		exptdBody []byte
-		cb        func(name, extraVars string) error
+		cb        func(name string, flags ActionFlags) error
 	}{
 		"commission": {
 			expURLStr: fmt.Sprintf("http://%s/%s/%s", baseURL, PostNodeCommissionPrefix, testNodeName),
 			nodeName:  testNodeName,
-			extraVars: "",
-			exptdBody: []byte{},
+			flags:     ActionFlags{"", ansibleMasterGroupName},
+			exptdBody: reqHostGroupBody.Bytes(),
```

Comment:
the client unit-tests validate that the body is set correctly when the respective go-client method is invoked. 

Accordingly we should add separate testcases for the body with the new host-group fields. We need to modify the `okReturner` and `failureReturner` above instead to make sure that whatever we fill in the host-group field is actually passed on correctly by the client.